bw_projects/project.py

```diff
@@ -7,17 +7,16 @@
 from threading import ThreadError
 
 import appdirs
-from bw_processing import safe_filename
 from peewee import BooleanField, DoesNotExist, Model, TextField
 
 from . import config
-from .filesystem import create_dir, maybe_path
-from .sqlite import PickleField, SubstitutableDatabase
+from .filesystem import create_dir, maybe_path, safe_filename
+from .sqlite import JSONField, SubstitutableDatabase
 
 
 class ProjectDataset(Model):
-    data = PickleField()
     name = TextField(index=True, unique=True)
+    attributes = JSONField()
```

Comment:
I like this idea, but we need to make sure that this is clear in the documentation, and is marked as a breaking change for existing Brightway users. I don't think it should effect many users, but better safe than sorry.